python_modules/dagster/dagster/_core/definitions/decorators/decorator_assets_definition_builder.py

```diff
@@ -543,20 +544,42 @@ def create_op_definition(self) -> OpDefinition:
         )(self.fn)
 
     def create_assets_definition(self) -> AssetsDefinition:
-        return AssetsDefinition.dagster_internal_init(
-            keys_by_input_name=self.asset_keys_by_input_names,
-            keys_by_output_name=self.asset_keys_by_output_name,
-            node_def=self.create_op_definition(),
-            partitions_def=self.args.partitions_def,
-            can_subset=self.args.can_subset,
-            resource_defs=self.args.assets_def_resource_defs,
-            backfill_policy=self.args.backfill_policy,
-            check_specs_by_output_name=self.check_specs_by_output_name,
-            specs=self.specs,
-            is_subset=False,
-            selected_asset_keys=None,  # not a subset so this is None
-            selected_asset_check_keys=None,  # not a subset so this is none
-        )
+        if len(self.specs) == 1 and len(self.check_specs_by_output_name) == 0:
```

Comment:
I'm not actually proposing we make this change. This is just a hack for getting wide test coverage on `AssetDefinition`, because it makes it get used basically in every test that uses `@asset`.